tests/conftest.py

```diff
@@ -42,7 +42,10 @@
     custom_cpu_templates_params,
     static_cpu_templates_params,
 )
-from framework.utils_drive import partuuid_and_disk_path_tmpfs
+from framework.utils_drive import (
+    partuuid_and_disk_path_host,
+    partuuid_and_disk_path_tmpfs,
```

Comment:
Same nit: since those are very specific fixtures, they could just be defined in the file that needs them. They can always be "promoted" later if they become useful for other tests.